app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -28,7 +38,24 @@ fun ConversationRouter(conversationListViewModel: ConversationListViewModel = hi
     }
 
     Scaffold(
-        floatingActionButton = { FloatingActionButton(stringResource(R.string.label_new), {}) },
+        floatingActionButton = {
+            FloatingActionButton(
+                text = stringResource(R.string.label_new),
+                icon = {
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_conversation),
+                        contentDescription = stringResource(R.string.content_description_new_conversation),
+                        contentScale = ContentScale.FillBounds,
+                        colorFilter = ColorFilter.tint(MaterialTheme.colorScheme.onPrimary),
+                        modifier = Modifier
+                            .padding(start = dimensions().spacing4x, top = dimensions().spacing2x)
```

Comment:
maybe I was not following the discussion, but what does this spacing2x, spacing4x mean ?

Reply:
basically, the idea here is to have such spaces in dimensions. At first I named it like `spacing2` which would be `2.dp` in regulare theme and whatever we need in others. But Gonso mentioned that such naming could be confusing, so I've added `x`.
So the idea here is to have names like: `spacing2x` where `spacing` - is just a name for the dimension, `2` - is 2.dp in regular theme `x` - is kinda multiplier sign. 
So if we decide that for tablets all the spacing should be doubled, we'll have `spacer2x = 4.dp`